lib/src/kernels/pooling/mli_krn_maxpool_hwc_decl.h

```diff
@@ -0,0 +1,116 @@
+/*
+* Copyright 2020-2020, Synopsys, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the BSD-3-Clause license found in
+* the LICENSE file in the root directory of this source tree.
+*
+*/
+
+#ifndef _MLI_KRN_MAXPOOL_DECL_REF_H_
```

Comment:
Should it be renamed to _MLI_KRN_MAXPOOL_DECL_H_?
Because in this file you declared ref and dsp versions of code.